docs/modules/operation/pages/deep-dive/admin/scv-configuration.adoc

```diff
@@ -1,52 +1,67 @@
+
 [[scv-overview]]
 = Secure Credentials Vault
 
-Secure Credentials Vault (SCV) lets you encrypt credentials the {page-component-title} system uses.
-SCV uses a Java KeyStore (JKS) to store credentials in JKS format.
-SCV is enabled by default on {page-component-title}.
+The secure credentials vault (SCV) lets you encrypt credentials used by {page-component-title}.
+It uses Java KeyStore (JKS) to store credentials in the JKS format.
+
+The SCV is enabled by default.
+
+== Configure SCV password
 
-== Configuring SCV password
+A custom SCV password is defined by the `org.opennms.features.scv.jceks.key` property in `$\{OPENNMS_HOME}/etc/opennms.properties.d/svc.properties`.
+The SCV keystore file is located at `$\{OPENNMS_HOME}/etc/scv.jce`.
 
-A custom SCV password can be defined with the `org.opennms.features.scv.jceks.key` system property in the `$\{OPENNMS_HOME}/etc/opennms.properties.d/svc.properties` file.
-The SCV keystore file is stored at `$\{OPENNMS_HOME}/etc/scv.jce`.
+IMPORTANT: The SCV password cannot be changed once credentials have been stored.
+`$\{OPENNMS_HOME}/etc/scv.jce` must be deleted and any credentials manually re-added if you want to change the password after the file has been created.
```

Comment:
```suggestion
You must delete `$\{OPENNMS_HOME}/etc/scv.jce` and manually re-add any credentials if you want to change the password after the file has been created.
```
To avoid passive voice, and also not start the sentence with a file name path.